README.md

```diff
@@ -245,11 +245,17 @@
 ### Term representation
 
 * <a name="underscores-in-numerics"></a>
-  Add underscores to large numeric literals to improve their readability.
+  Add underscores to base-10 numeric literals that have six or more digits.
```

Comment:
I wouldn't write a port with the underscore in it because that's not how I would write it in system env variables, in a blog post, or wherever else. I think 6 or more digits is a good compromise because of this.